egs/callhome_egyptian/s5/RESULTS

```diff
@@ -166,3 +166,61 @@ exp/tri6a_dnn/decode_h5/wer_15 %WER 59.32 [ 9957 / 16785, 284 ins, 3960 del, 571
+exp/nnet3/nnet_tdnn_a/decode_dev/wer_7 %WER 54.44 [ 18242 / 33509, 2092 ins, 3714 del, 12436 sub ]
+exp/nnet3/nnet_tdnn_a/decode_dev/wer_8 %WER 53.63 [ 17970 / 33509, 1766 ins, 4087 del, 12117 sub ]
+exp/nnet3/nnet_tdnn_a/decode_dev/wer_9 %WER 53.51 [ 17931 / 33509, 1533 ins, 4452 del, 11946 sub ]
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+*** Test ***
+exp/nnet3/nnet_tdnn_a/decode_test/wer_10 %WER 52.29 [ 8177 / 15637, 536 ins, 2190 del, 5451 sub ]
+exp/nnet3/nnet_tdnn_a/decode_test/wer_11 %WER 52.49 [ 8208 / 15637, 474 ins, 2373 del, 5361 sub ]
+exp/nnet3/nnet_tdnn_a/decode_test/wer_12 %WER 53.03 [ 8293 / 15637, 420 ins, 2558 del, 5315 sub ]
+exp/nnet3/nnet_tdnn_a/decode_test/wer_13 %WER 53.52 [ 8369 / 15637, 361 ins, 2721 del, 5287 sub ]
+exp/nnet3/nnet_tdnn_a/decode_test/wer_2 %WER 64.18 [ 10036 / 15637, 2048 ins, 980 del, 7008 sub ]
+exp/nnet3/nnet_tdnn_a/decode_test/wer_3 %WER 60.84 [ 9513 / 15637, 1726 ins, 1076 del, 6711 sub ]
+exp/nnet3/nnet_tdnn_a/decode_test/wer_4 %WER 58.23 [ 9106 / 15637, 1471 ins, 1210 del, 6425 sub ]
+exp/nnet3/nnet_tdnn_a/decode_test/wer_5 %WER 55.77 [ 8720 / 15637, 1206 ins, 1351 del, 6163 sub ]
+exp/nnet3/nnet_tdnn_a/decode_test/wer_6 %WER 54.19 [ 8474 / 15637, 1005 ins, 1505 del, 5964 sub ]
+exp/nnet3/nnet_tdnn_a/decode_test/wer_7 %WER 53.08 [ 8300 / 15637, 827 ins, 1689 del, 5784 sub ]
+exp/nnet3/nnet_tdnn_a/decode_test/wer_8 %WER 52.43 [ 8198 / 15637, 712 ins, 1841 del, 5645 sub ]
+exp/nnet3/nnet_tdnn_a/decode_test/wer_9 %WER 52.10 [ 8147 / 15637, 619 ins, 1993 del, 5535 sub ]
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+*** Supplement ***
+exp/nnet3/nnet_tdnn_a/decode_sup/wer_10 %WER 80.31 [ 13981 / 17409, 407 ins, 8528 del, 5046 sub ]
+exp/nnet3/nnet_tdnn_a/decode_sup/wer_11 %WER 80.42 [ 14001 / 17409, 360 ins, 8752 del, 4889 sub ]
+exp/nnet3/nnet_tdnn_a/decode_sup/wer_12 %WER 80.52 [ 14017 / 17409, 318 ins, 8968 del, 4731 sub ]
+exp/nnet3/nnet_tdnn_a/decode_sup/wer_13 %WER 80.79 [ 14065 / 17409, 291 ins, 9155 del, 4619 sub ]
+exp/nnet3/nnet_tdnn_a/decode_sup/wer_2 %WER 85.93 [ 14960 / 17409, 1330 ins, 6454 del, 7176 sub ]
+exp/nnet3/nnet_tdnn_a/decode_sup/wer_3 %WER 84.65 [ 14737 / 17409, 1151 ins, 6635 del, 6951 sub ]
+exp/nnet3/nnet_tdnn_a/decode_sup/wer_4 %WER 83.31 [ 14504 / 17409, 968 ins, 6890 del, 6646 sub ]
+exp/nnet3/nnet_tdnn_a/decode_sup/wer_5 %WER 82.52 [ 14366 / 17409, 839 ins, 7159 del, 6368 sub ]
+exp/nnet3/nnet_tdnn_a/decode_sup/wer_6 %WER 81.66 [ 14216 / 17409, 711 ins, 7477 del, 6028 sub ]
+exp/nnet3/nnet_tdnn_a/decode_sup/wer_7 %WER 81.08 [ 14116 / 17409, 631 ins, 7750 del, 5735 sub ]
+exp/nnet3/nnet_tdnn_a/decode_sup/wer_8 %WER 80.52 [ 14017 / 17409, 547 ins, 7999 del, 5471 sub ]
+exp/nnet3/nnet_tdnn_a/decode_sup/wer_9 %WER 80.31 [ 13982 / 17409, 468 ins, 8269 del, 5245 sub ]
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
```

Comment:
That should help.  What would be even better is to train with training data
with perturbed volumes (hopefully including the range of the actual test
data).
The online-nnet2 system is supposed to learn the correct normalization from
the iVector, but it can't do so if there was no variation in that dimension
in the training data, i.e. if the training data was all normalized to the
same volume.
Dan


On Mon, Oct 12, 2015 at 10:36 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/callhome_egyptian/s5/RESULTS
> <https://github.com/kaldi-asr/kaldi/pull/238#discussion_r41821901>:
>
> > +exp/nnet3/nnet_tdnn_a/decode_test/wer_9 %WER 52.10 [ 8147 / 15637, 619 ins, 1993 del, 5535 sub ]
> > +@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
> > +*** Supplement ***
> > +exp/nnet3/nnet_tdnn_a/decode_sup/wer_10 %WER 80.31 [ 13981 / 17409, 407 ins, 8528 del, 5046 sub ]
> > +exp/nnet3/nnet_tdnn_a/decode_sup/wer_11 %WER 80.42 [ 14001 / 17409, 360 ins, 8752 del, 4889 sub ]
> > +exp/nnet3/nnet_tdnn_a/decode_sup/wer_12 %WER 80.52 [ 14017 / 17409, 318 ins, 8968 del, 4731 sub ]
> > +exp/nnet3/nnet_tdnn_a/decode_sup/wer_13 %WER 80.79 [ 14065 / 17409, 291 ins, 9155 del, 4619 sub ]
> > +exp/nnet3/nnet_tdnn_a/decode_sup/wer_2 %WER 85.93 [ 14960 / 17409, 1330 ins, 6454 del, 7176 sub ]
> > +exp/nnet3/nnet_tdnn_a/decode_sup/wer_3 %WER 84.65 [ 14737 / 17409, 1151 ins, 6635 del, 6951 sub ]
> > +exp/nnet3/nnet_tdnn_a/decode_sup/wer_4 %WER 83.31 [ 14504 / 17409, 968 ins, 6890 del, 6646 sub ]
> > +exp/nnet3/nnet_tdnn_a/decode_sup/wer_5 %WER 82.52 [ 14366 / 17409, 839 ins, 7159 del, 6368 sub ]
> > +exp/nnet3/nnet_tdnn_a/decode_sup/wer_6 %WER 81.66 [ 14216 / 17409, 711 ins, 7477 del, 6028 sub ]
> > +exp/nnet3/nnet_tdnn_a/decode_sup/wer_7 %WER 81.08 [ 14116 / 17409, 631 ins, 7750 del, 5735 sub ]
> > +exp/nnet3/nnet_tdnn_a/decode_sup/wer_8 %WER 80.52 [ 14017 / 17409, 547 ins, 7999 del, 5471 sub ]
> > +exp/nnet3/nnet_tdnn_a/decode_sup/wer_9 %WER 80.31 [ 13982 / 17409, 468 ins, 8269 del, 5245 sub ]
> > +@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
>
> You could decode with the current models, after normalizing the test data
> to be in the training data range.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/238/files#r41821901>.
>